internal/engine/interpreter/interpreter.go

```diff
@@ -459,143 +511,7 @@ func (e *engine) lowerIR(ir *wazeroir.CompilationResult) (*code, error) {
 			op.U2 = uint64(o.TableIndex)
 		case wazeroir.OperationElemDrop:
 			op.U1 = uint64(o.ElemIndex)
-		case wazeroir.OperationTableCopy:
-			op.U1 = uint64(o.SrcTableIndex)
-			op.U2 = uint64(o.DstTableIndex)
-		case wazeroir.OperationRefFunc:
-			op.U1 = uint64(o.FunctionIndex)
-		case wazeroir.OperationTableGet:
-			op.U1 = uint64(o.TableIndex)
-		case wazeroir.OperationTableSet:
-			op.U1 = uint64(o.TableIndex)
-		case wazeroir.OperationTableSize:
-			op.U1 = uint64(o.TableIndex)
-		case wazeroir.OperationTableGrow:
-			op.U1 = uint64(o.TableIndex)
-		case wazeroir.OperationTableFill:
-			op.U1 = uint64(o.TableIndex)
-		case wazeroir.OperationV128Const:
-			op.U1 = o.Lo
-			op.U2 = o.Hi
-		case wazeroir.OperationV128Add:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Sub:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Load:
-			op.B1 = o.Type
-			op.U1 = uint64(o.Arg.Alignment)
-			op.U2 = uint64(o.Arg.Offset)
-		case wazeroir.OperationV128LoadLane:
-			op.B1 = o.LaneSize
-			op.B2 = o.LaneIndex
-			op.U1 = uint64(o.Arg.Alignment)
-			op.U2 = uint64(o.Arg.Offset)
-		case wazeroir.OperationV128Store:
-			op.U1 = uint64(o.Arg.Alignment)
-			op.U2 = uint64(o.Arg.Offset)
-		case wazeroir.OperationV128StoreLane:
-			op.B1 = o.LaneSize
-			op.B2 = o.LaneIndex
-			op.U1 = uint64(o.Arg.Alignment)
-			op.U2 = uint64(o.Arg.Offset)
-		case wazeroir.OperationV128ExtractLane:
-			op.B1 = o.Shape
-			op.B2 = o.LaneIndex
-			op.B3 = o.Signed
-		case wazeroir.OperationV128ReplaceLane:
-			op.B1 = o.Shape
-			op.B2 = o.LaneIndex
-		case wazeroir.OperationV128Splat:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Shuffle:
-			op.Us = make([]uint64, 16)
-			for i, l := range o.Lanes {
-				op.Us[i] = uint64(l)
-			}
-		case wazeroir.OperationV128Swizzle:
-		case wazeroir.OperationV128AnyTrue:
-		case wazeroir.OperationV128AllTrue:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128BitMask:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128And:
-		case wazeroir.OperationV128Not:
-		case wazeroir.OperationV128Or:
-		case wazeroir.OperationV128Xor:
-		case wazeroir.OperationV128Bitselect:
-		case wazeroir.OperationV128AndNot:
-		case wazeroir.OperationV128Shr:
-			op.B1 = o.Shape
-			op.B3 = o.Signed
-		case wazeroir.OperationV128Shl:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Cmp:
-			op.B1 = o.Type
-		case wazeroir.OperationV128AddSat:
-			op.B1 = o.Shape
-			op.B3 = o.Signed
-		case wazeroir.OperationV128SubSat:
-			op.B1 = o.Shape
-			op.B3 = o.Signed
-		case wazeroir.OperationV128Mul:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Div:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Neg:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Sqrt:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Abs:
-			op.B1 = o.Shape
-		case wazeroir.OperationV128Popcnt:
```

Comment:
apparently Shape is actually being ignored everywhere for this Op. Is this correct? (I assume so, just double-checking)